ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -886,13 +886,31 @@ class EclCmdRun : public EclCmdWithEclTarget
 
         switch (state)
         {
-        case WUStateRunning:
-            fprintf(stderr, "Timed out waiting for %s to complete, workunit is still running.\n", wuid.str()); //server side waiting timed out
-            break;
         case WUStateCompleted:
             break;
+        case WUStateCompiled:
```

Comment:
The deploy command currently returns 1 if the following is not true:

bool isCompiled = (strieq(state, "compiled")||strieq(state, "completed"));